File: uber-clone/src/main/java/com/backend/uberclone/model/Note.java
package com.backend.uberclone.model;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Note {

        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        private Integer id;

        @ManyToOne(fetch = FetchType.EAGER)
        @JoinColumn(name="notes")
        private User user;

        @Column
        private String text;

        @Column
        private LocalDateTime date;


}

File: front/uber-clone/src/app/models/note.ts
export class Note {
  id: number;
  text: string;
  date: Date;
  user: number;

  constructor(id: number, text: string, date: Date, user: number) {
    this.id = id;
    this.text = text;
    this.date = date;
    this.user = user;
  }
}
